CMSgov/mdct-mcr
# Section B: Program Integrity (ii)

## Description

Adding nested fields for this section.

### How to test

- `./dev local`
- Navigate to /mcpar/state-level-indicators/program-integrity
- See fields!

### Changed Dependencies

N/A 

## Code author checklist

## Reviewer checklist (two different people)
